dockerfiles/prod.Dockerfile

```
@@ -30,6 +30,10 @@ FROM alpine:3.18.2
 
 WORKDIR /
 
+RUN apk add --update --no-cache ca-certificates tzdata && update-ca-certificates \
+    && cp /usr/share/zoneinfo/Asia/Tokyo /etc/localtime \
+    && rm -rf /usr/share/zoneinfo
```

この書き方にしたら`time.LoadLocation("Asia/Tokyo")`がちゃんと実行できました
```suggestion
RUN apk add --update --no-cache tzdata \
    && cp /usr/share/zoneinfo/Asia/Tokyo /etc/localtime
```

portalの実装ではタイムゾーンの参照は`Local`としてしかしていないので`/etc/localtime`にさえ`Asia/Tokyo`の情報があればOKだったみたいです

ですが、traQ gazerでは`time.LoadLocation("Asia/Tokyo")`として参照しているので`/usr/share/zoneinfo`以下に該当のタイムゾーンの情報が存在している必要があるみたいです
[golangのtimeライブラリでタイムゾーン変換](https://blog.freedom-man.com/golang-time-zone)

また、`ca-certificates`パッケージやそれを使っての`update-ca-certificates`ではtraQ gazerには不要です